x-pack/plugins/alerting/server/authorization/audit_logger.ts

```diff
@@ -29,9 +29,10 @@ export class AlertsAuthorizationAuditLogger {
     alertTypeId: string,
     scopeType: ScopeType,
     scope: string,
-    operation: string
+    operation: string,
+    authorizationType: string
   ): string {
-    return `${authorizationResult} to ${operation} a "${alertTypeId}" alert ${
+    return `${authorizationResult} to ${operation} a "${alertTypeId}" ${authorizationType} ${
```

Comment:
One thought that crosses my mind - once customers upgrade to 7.14 their audit logs will say `rule` where they used to say `alert`, and `alert` on the new Alert entities.
This means that if a user looks at their audit logs across versions they might have two audit log rows that say Alert, but in one its a Rule and in the other an Alert.

I'm not sure what we can do about that, but perhaps it's worth making it easier to distinguish between them by changing the format in 7.14 so that the sentences at least appear different in some manner? 🤔